File: FAOSolution/src/FAO.DAL/Entities/Asset.cs
using System;
using System.Collections.Generic;

namespace FAO.DAL.Entities
{
    public class Asset
    {
        public Guid TenantId { get; set; }
        public Guid CompanyId { get; set; }
        public int AssetId { get; set; }
        public string PropType { get; set; }
        public string Description { get; set; }
        public string Location { get; set; }
        public string Department { get; set; }

        public List<Assetevent> Assetevents { get; set; }
        public List<Assetimage> Assetimages { get; set; }
        public List<Partialinfo> Partialinfos { get; set; }

        public Company Company { get; set; }
    }
}

File: FAOSolution/src/FAO.APP.WebSite/ClientApp/app/models/asset.ts
export class Asset {
    tenantId: string;
    companyId: string;
    assetId: number;
    propType: string;
    description: string;
    location: string;
    department: string;

    constructor(attributes: {} = null) {
        this.attributes = attributes;
    }

    set attributes(attributes: {}) {
        for (var k in attributes) {
            this[k] = attributes[k];
        }
    }

    get attributes(): {} {
        return {
            tenantId: this.tenantId,
            companyId: this.companyId,
            assetId: this.assetId,
            propType: this.propType,
            description: this.description,
            location: this.location,
            department: this.department
        };
    }

}